pkg/elastic/client.go

```diff
@@ -135,6 +146,47 @@ func (c *Client) UpdateUser(ctx context.Context, username string, user Deploymen
 	return nil
 }
 
+// CreateUserMappingRole creates mapping role.
+func (c *Client) CreateUserMappingRole(ctx context.Context, body MappingRolesBody, name string) error {
+	url, _ := url.JoinPath(c.deploymentEndpoint, "_security/role_mapping/"+name)
```

Comment:
Instead of concatenating strings, name should be another argument to url.joinpath. Same goes for the constructing the url in AddDeploymentRoleMapping

Reply:
Fixed